### ISSUE TITLE
[ANDROID] getCurrentUser java.lang.SecurityException on Samsung S8 Oreo

### ISSUE THREAD

**Issue by TFreudi1** [xamarin/Xamarin.Forms#3182]
I build a simple PCL Solution, with the wizard and changed nothing. In the Android part after a while the debug window shows: 

java.lang.SecurityException: Permission Denial: getCurrentUser() from pid=24983, uid=10498 requires android.permission.INTERACT_ACROSS_USERS

I use VS 2017 15.7.4 Xamarin.Forms 3.1.0.583944 and it only happens on my S8 Oreo 8.0, i tried different Emulators and other Android mobiles, it happens only on the S8, but through this continous messages in the log-window I can't see the realy important messages.

Full Stacktrace:
java.lang.SecurityException: Permission Denial: getCurrentUser() from pid=27837, uid=10504 requires android.permission.INTERACT_ACROSS_USERS
06-29 10:48:00.441 W/System.err(27837): 	at android.os.Parcel.readException(Parcel.java:1958)
06-29 10:48:00.441 W/System.err(27837): 	at android.os.Parcel.readException(Parcel.java:1904)
06-29 10:48:00.441 W/System.err(27837): 	at android.sec.clipboard.IClipboardService$Stub$Proxy.isEnabled(IClipboardService.java:757)
06-29 10:48:00.441 W/System.err(27837): 	at com.samsung.android.content.clipboard.SemClipboardManager.isEnabled(SemClipboardManager.java:1008)
06-29 10:48:00.441 W/System.err(27837): 	at android.widget.TextView.getSecClipboardEnabled(TextView.java:16133)
06-29 10:48:00.441 W/System.err(27837): 	at android.widget.TextView.onDetachedFromWindowInternal(TextView.java:7178)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.View.dispatchDetachedFromWindow(View.java:18659)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3878)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3878)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.ViewGroup.removeViewInternal(ViewGroup.java:5432)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.ViewGroup.removeViewInternal(ViewGroup.java:5403)
06-29 10:48:00.442 W/System.err(27837): 	at android.view.ViewGroup.removeView(ViewGroup.java:5334)
06-29 10:48:00.442 W/System.err(27837): 	at mono.java.lang.RunnableImplementor.n_run(Native Method)
06-29 10:48:00.442 W/System.err(27837): 	at mono.java.lang.RunnableImplementor.run(RunnableImplementor.java:30)
06-29 10:48:00.442 W/System.err(27837): 	at android.os.Handler.handleCallback(Handler.java:789)
06-29 10:48:00.442 W/System.err(27837): 	at android.os.Handler.dispatchMessage(Handler.java:98)
06-29 10:48:00.442 W/System.err(27837): 	at android.os.Looper.loop(Looper.java:164)
06-29 10:48:00.442 W/System.err(27837): 	at android.app.ActivityThread.main(ActivityThread.java:6938)
06-29 10:48:00.442 W/System.err(27837): 	at java.lang.reflect.Method.invoke(Native Method)
06-29 10:48:00.442 W/System.err(27837): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
06-29 10:48:00.443 W/System.err(27837): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)

**Comment by hartez** [comment]
Are you seeing this in the Visual Studio output, or in the Android device log? Is it always the same process id? If you're seeing it in the Android device log, have you tried filtering out the process id which is logging the error?

**Comment by TFreudi1** [comment]
![xamarin_security](https://user-images.githubusercontent.com/4739177/42378867-1df77c48-8129-11e8-9b36-2384520a772c.png)
It is the VS Output. The PID is always different.

**Comment by PureWeen** [comment]
@TFreudi1  can you attach your project to this issue? I created a blank project and I'm running it on an s8 and not seeing the same exceptions you are

From the looks of the exception it seems like it's trying to pop open a permissions dialog

Maybe also make sure all your android sdk and build tools are updated

**Comment by TFreudi1** [comment]
You will get it tomorrow, i guess a One-Drive link would be OK ?

**Comment by PureWeen** [comment]
yes

**Comment by TFreudi1** [comment]
Hmm, at the Moment none of my projects Show this error, I must check what it makes happen ...

**Comment by TFreudi1** [comment]
Sorry PureWeen my main Job is programming in C, only few additonal stuff is written with Xamarin.
I updated my Xamarin-projects (one after another) to Xamarin.Forms.3.1.637273 and the Messages in all a projects gone.
I did'nt change anything on the mobile. I will close this issue and open a new on if it's reappearing, and if thats the case i will put a sample Project here. Thanks for your help so far !
